src/Microsoft.Azure.WebJobs.Extensions.Kafka/Microsoft.Azure.WebJobs.Extensions.Kafka.csproj

```diff
@@ -7,6 +7,16 @@
     <RootNamespace>Microsoft.Azure.WebJobs.Extensions.Kafka</RootNamespace>
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Kafka</PackageId>
     <Description>Microsoft Azure WebJobs SDK Kafka Extension</Description>
+    <CommitHash Condition="$(CommitHash) == ''">N/A</CommitHash>
+    <InformationalVersion>$(Version) Commit hash: $(CommitHash)</InformationalVersion>
+    <Authors>Microsoft</Authors>
+    <Company>Microsoft</Company>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseUrl>http://www.microsoft.com/web/webpi/eula/aspnetcomponent_rtw_enu.htm</PackageLicenseUrl>
+    <PackageProjectUrl>http://go.microsoft.com/fwlink/?LinkID=320972</PackageProjectUrl>
```

Comment:
Should we be pointing to the WebJobs SDK? Why aren't we pointing to our project's repo? 
@paulbatum & @fabiocav  is this a convention the other extensions all follow?

Reply:
I see there is another PackageRepositoryUrl below and that does in fact point to our repo. So if the intention of PackageProjectUrl is to point to WebJobs because this is part of that "Package", then I am ok with the current url being used.

Reply:
Thanks. Wait for their review. :)

Reply:
I think we're fine. I checked a few others and they all look the same. So let's stick with this and we can change it later if needed.